include/ofi_util.h

```diff
@@ -554,6 +554,63 @@ int ofi_cq_write_overflow(struct util_cq *cq, void *context, uint64_t flags,
 			  size_t len, void *buf, uint64_t data, uint64_t tag,
 			  fi_addr_t src);
 
+static inline
+ssize_t ofi_cq_read_cq(struct util_cq *cq, void *buf, size_t count,
+			fi_addr_t *src_addr)
```

Comment:
it seems the function mainly flush the cirque, can we name it `ofi_cq_flush_cirque` to be more specific?

Reply:
If this is going to be implemented by SM2, there is no circular queue... I think cq is overloaded for completion queue

Reply:
Done.

Reply:
I disagree here.  In other places in the code, 'flush' means we're discarding the entries.  That's not the case here.  This is actually reading from the cq.  The behavior is closer to ofi_cq_read_entries(), if we match up with the naming for ofi_cq_write_entry.

Reply:
Thanks, updated the name.